extensions/interactions/MultipleChoiceInput/directives/multiple-choice-input-validation.service.ts

```diff
@@ -77,7 +75,8 @@ export class MultipleChoiceInputValidationService {
   }
 
   getAllWarnings(
-      stateName: string, customizationArgs: any, answerGroups: AnswerGroup[],
+      stateName: string, customizationArgs:
```

Comment:
Keep parameters and their types on the same line.

Reply:
done